pkg/filterengine/filterengine.go

```diff
@@ -10,6 +10,11 @@ import (
 	"github.com/kubeshop/botkube/pkg/maputil"
```

Comment:
actually this file can be deleted, and in the test we should use `filters.DisableAnnotation` from k8s source. I will handle that on my PR, don't worry about that.